pkgs/tools/system/htop/default.nix

```diff
@@ -1,19 +1,23 @@
 { lib, fetchurl, stdenv, ncurses,
-IOKit }:
+IOKit, python }:
 
 stdenv.mkDerivation rec {
   name = "htop-${version}";
-  version = "2.0.2";
+  version = "2.1.0";
 
   src = fetchurl {
-    sha256 = "11zlwadm6dpkrlfvf3z3xll26yyffa7qrxd1w72y1kl0rgffk6qp";
     url = "http://hisham.hm/htop/releases/${version}/${name}.tar.gz";
+    sha256 = "0j07z0xm2gj1vzvbgh4323k4db9mr7drd7gw95mmpqi61ncvwq1j";
   };
 
   buildInputs =
-    [ ncurses ] ++
+    [ ncurses python ] ++
     lib.optionals stdenv.isDarwin [ IOKit ];
 
+  patchPhase = ''
```

Comment:
I think it's better to use `post/prePatch`.

Reply:
Yes, either is fine. I always tend to use `prePatch`. You should usually not touch the `patchPhase` unless really required. If we ever need the standard `patches` attribute with this derivation not having to rename other attributes would be great :-)